saleor/checkout/models.py

```diff
@@ -104,14 +106,31 @@ def get_subtotal(self, discounts=None, taxes=None):
         subtotals = (line.get_total(discounts, taxes) for line in self)
         return sum(subtotals, ZERO_TAXED_MONEY)
 
-    def get_total(self, discounts=None, taxes=None):
-        """Return the total cost of the checkout."""
+    def get_total_without_gift_cards(self, discounts=None, taxes=None):
```

Comment:
I'm wondering if we really need it. It would be great to have just `get_total` as it was, so you don't have to think which of these two methods you should use.

Reply:
(But that's just a loose thought, maybe there is no other way)

Reply:
I'm pretty sure that this method will be removed after implementing gift cards with the payment method. 
If I have enough time, I will remove this method and calculate these values where I need them.